lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -512,56 +512,56 @@ void forceDeleteTask(String taskId)
       throws InvalidArgumentException;
 
   /**
-   * Updates tasks with matching taskIds.
+   * Updates {@link Task}s with matching taskIds.
```

Comment:
"Update custom fields for all given Tasks"?

Reply:
What do you think about "Updates specified {@linkplain TaskCustomField TaskCustomFields} for all given {@linkplain Task
   Tasks}" ?